docs/quickstart.md

```diff
@@ -21,6 +21,15 @@ docker run -p 30001:30001 -p 30002:30002 -p 30003:30003 redislabs/rgcluster:late
 
 ## Building
 
+### Clone
+To get the code and its submodules do the following:
+```
+git clone https://github.com/RedisGears/RedisGears.git
```

Comment:
`git clone --recursive https://github.com/RedisGears/RedisGears.git` in one step.